config/google2fa.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+return [
+
+    /*
+     * Enable / disable Google2FA.
+     */
+    'enabled' => env('OTP_ENABLED', true),
```

Comment:
Oh, I think our own `APP_2FA` is useless, if this has already a config for it. You can remove `APP_2FA` and replace this config's defualt iwth `false` to do the exact same thing (and remove the code checking for `2fa_enabled` in the app, checking `google2fa.enabled` instead.